package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "fipeTable",
+  "version": "1.0.0",
+  "description": "",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
provide a script here to build using webpack. Something that I can run like: `npm run build` and the final result will be the bundle.js